src/main/java/com/eharmony/pho/api/DataStoreApi.java

```diff
@@ -72,4 +74,13 @@
      */
     <T, R> R findOne(QuerySelect<T, R> query);
 
+    /**
+     * Save selective field for the given entity
```

Comment:
Clarify the JavaDoc to indicate that this updates an existing entity, but only for the selected fields.